testing/test_capture.py

```diff
@@ -1112,6 +1136,22 @@ def test_stdin_nulled_by_default(self):
             pytest.raises(IOError, sys.stdin.read)
 
 
+class TestTeeStdCapture(TestStdCapture):
```

Comment:
Good tests!

I suggest we add a test to `acceptance_tests.py` which uses `--capture=tee-stdio` and `--junitxml` and checks that output is shown in the terminal and in the generated XML file, just to ensure we don't regress the original intent of the feature.

Reply:
Great idea! Added one with a045e3f52dcca.